doc/ug/dv_methodology/index.md

```diff
@@ -299,61 +299,116 @@ Once the regression tool is developed, it will provide a way to enable these cap
-They are sampled in 'reactive' components of the testbench, such as monitors and / or the scoreboards.
+Unlike code coverage, functional coverage requires the designer and/or DV engineer to write additional cover points and covergroups.
+For this reason functional coverage is sometimes referred to as explicit coverage.
+These are more complex constructs that capture whether signals (that reflect the current state of the design) have met an interesting set or a sequence of values (often called corner cases)
+These constructs also allow us to capture whether multiple scenarios have occurred simultaneously through crosses.
+This is also often referred to as cross coverage.
+These constructs are typically encapsulated in a class and are sometimes referred to as the 'functional coverage model'.
+They are sampled in 'reactive' components of the testbench, such as monitors and/or the scoreboards.
 They can also be embedded within the RTL to sample interesting scenarios through DV stimulus.
 
 Here are the metrics used with a brief explanation:
 
-*  **Assert Coverage**: This metric measures which assertions, cover properties and sequences have been exercised in the course of the simulation.
-  Note, an assertion precondition counts as a cover point.
-*  **Covergroup Coverage**: This metric (sometimes called "Testbench Coverage") measures covergroups during simulation.
-  These are usually coded in the testbench to check that the desired stimulus was generated properly, but can also be embedded in the RTL.
+* **Covergroup Coverage**: Observes values on buses, registers and so on.
+  This can verify that a specific value or range of values was seen on a bus and that no illegal values were seen.
+  Simulators can be set to throw an error if an illegal value is seen.
+  Another example of a covergroup could be the intersection of two cover points.
+  One example could be observing the FIFO full signal and the FIFO write enable signal is asserted at the same time.
+  This is called a coverage cross.
+* **Cover Property coverage**:
+   * **Assertion coverage using SVA**
+   * **procedural code**
+
+Most often Property coverage is implemented using Systemverilog Assertions (SVA).
+This observes events or series of events.
+As an example think of the TL UL register bus used in OpenTitan.
+A cover property cover point could be the handshaking between valid and ready.
+SVA also allows the Design engineer to add cover procedures and variables not visible on output pins.
+Note, an assertion precondition counts as a cover point.
+
+
+#### Do we need both types of coverage
+
+Reaching a 100% code coverage does not tell you the whole story.
+Think of the simple example of an AND gate with inputs A, B, and output O.
+To get to 100% code coverage only two different input combinations are needed.
+00 and 11, these two will produce all possible outcomes of O.
```

Comment:
The second sentence doesn't really work because it starts with a number (and there is a missing verb in the first clause). The simplest fix is probably: "To get to 100% code coverage only two different input combinations are needed: 00 and 11. These two combinations will produce all possible values of O."

Reply:
thank you, I was not aware that a sentence cannot start with numbers

Reply:
I'm not sure if it's a hard-and-fast rule, but it looks a bit weird and I don't think I've ever seen it in print.